### PR TITLE
Feat: v3.1.0 merge to main

### PR DESCRIPTION
**Pull request by brian-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [x] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [x] All commented code has been removed
- [x] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

Merge v3.1.0 changes to main

## Issue this PR will close

close: #**issue_number**

### CODE REVIEW

File: src/pipeline-server/docker-compose.pipeline-server.yml [hunk 1]
```diff
@@ -14,13 +14,13 @@ services:
...
     ports:
       - "8555:8554"
     environment:
-      - http_proxy=
-      - https_proxy=
-      - HTTP_PROXY=
-      - HTTPS_PROXY=
+      http_proxy: ${http_proxy}
+      https_proxy: ${https_proxy}
+      no_proxy: ${no_proxy}
```
Comment (antoniomtz) [inline src/pipeline-server/docker-compose.pipeline-server.yml:23]: I found an issue when setting proxies like this. The correct way is:

```suggestion
      - HTTP_PROXY
      - HTTPS_PROXY
      - NO_PROXY
```

File: src/pipeline-server/Dockerfile.pipeline-server [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-FROM intel/dlstreamer-pipeline-server:2022.2.0-ubuntu20-gpu419.40
+FROM intel/edge_video_analytics_microservice:2.0.0
```
Comment (antoniomtz) [inline src/pipeline-server/Dockerfile.pipeline-server:7]: I didn't find this public image on dockerhub yet. How are we supposed to download it ? https://hub.docker.com/r/intel/edge_video_analytics_microservice/tags

File: src/pipeline-server/docker-compose.pipeline-server.yml [hunk 3]
```diff
@@ -63,107 +63,53 @@ services:
...
+    container_name: pipeline-init
     command: run /postman/pipeline.json -e "/postman/env.json" --verbose
     environment:
-      - http_proxy=
-      - https_proxy=
-      - HTTP_PROXY=
-      - HTTPS_PROXY=
+      http_proxy: ${http_proxy}
+      https_proxy: ${https_proxy}
+      no_proxy: ${no_proxy}
```
Comment (antoniomtz) [inline src/pipeline-server/docker-compose.pipeline-server.yml:109]:
```suggestion
      - HTTP_PROXY
      - HTTPS_PROXY
      - NO_PROXY
```

File: src/pipeline-server/docker-compose.pipeline-server.yml [hunk 4]
```diff
@@ -250,12 +195,12 @@ services:
...
-      HOST_IP: evam_0
+      HOST_IP: ${HOST_IP}
       MODE: "EVA"
-      http_proxy:
-      https_proxy:
-      no_proxy:
-      GRAFANA_SERVER: 0.0.0.0
+      http_proxy: ${http_proxy}
+      https_proxy: ${https_proxy}
+      no_proxy: ${no_proxy}
```
Comment (antoniomtz) [inline src/pipeline-server/docker-compose.pipeline-server.yml:202]:
```suggestion
      - HTTP_PROXY
      - HTTPS_PROXY
      - NO_PROXY
```

File: src/pipeline-server/docker-compose.pipeline-server.yml [hunk 5]
```diff
@@ -264,23 +209,21 @@ services:
...
+    image: intel/simple-signaling-server:4.0.3
     container_name: webrtc-signaling-server
     read_only: true
     environment:
-      http_proxy:
-      https_proxy:
-      no_proxy:
+      http_proxy: ${http_proxy}
+      https_proxy: ${https_proxy}
+      no_proxy: ${no_proxy}
```
Comment (antoniomtz) [inline src/pipeline-server/docker-compose.pipeline-server.yml:222]:
```suggestion
      - HTTP_PROXY
      - HTTPS_PROXY
      - NO_PROXY
```

**Review by antoniomtz — CHANGES_REQUESTED** [review]

**Review by NeethuES-intel — APPROVED** [review]

**Review by jim-wang-intel — APPROVED** [review]

**Review by antoniomtz — APPROVED** [review]

**Pull request merged**